src/tests/itests/xi_itest_sft.c

```diff
@@ -153,7 +153,15 @@ void xi_itest_sft__remove_files( const char** filenames, uint16_t files_count )
     uint16_t id_file = 0;
     for ( ; id_file < files_count; ++id_file )
     {
+        /* delete files generated during tests (files received from mock broker) */
         xi_bsp_io_fs_remove( filenames[id_file] );
+
+#if 1
+        /* delete files containing revisionstrings */
```

Comment:
unfortunately you are right. I've left it there intentionally since it is good to see tests using the previous run's saved revision. But yeah, I'll create tests to cover checking the .xirev files. So yes, I'll remove this, thx.